server/config.js

```diff
@@ -0,0 +1,5 @@
+
+const url = 'http://localhost:3001';

```

Comment:
Not sure if you meant to gitignore config.js.

Reply:
it is not doing anything actually, planning to remove or change.